### DISCUSSION TITLE
Add startpage and brave search as search engines

### DISCUSSION THREAD

**Discussion by androidghosh** [bromite/bromite#1614] [Ideas]
Adding of https://startpage.com/ and https://search.brave.com/ as alternative search engines with autocomplete search query support.

**Comment by uazo** [comment]
you should already be able to do this on your own, see here https://github.com/bromite/bromite/wiki/SearchEngines

**Comment by chrislogan-foss** [comment]
> Bromite does not add* or remove any search engine to the pre-defined list available in Chromium; no new search engines will be added and there has never been nor will be any sponsorship/partnership with any search engine company.
